pkg/actions/import.go

```diff
@@ -248,15 +248,15 @@ func (i *Import) createYAML(fileName, base, ext string) error {
 func (i *Import) createComponentFromData(name, data string, templateType prototype.TemplateType) error {
 	componentParams := params.Params{}
 
+	var moduleName string
 	switch i.module {
```

Comment:
why keep this a switch when there's only one case? is a default-only switch a common golang idiom?

Reply:
1. Because 0 values are useful in Go. It's the same thing as:

```go
var moduleName
if i.module != "" && i.module != "/" {
  moduleName = i.module
}
```
2. It's not a default only switch. There is a case statement in there too.